shared/src/main/java/gov/hhs/cdc/trustedintermediary/external/hapi/HapiHelper.java

```diff
@@ -395,6 +396,19 @@ public static String getORC21Value(ServiceRequest serviceRequest) {
         return orc21Extension.getValue().primitiveValue();
     }
 
+    // OBR - Observation Request
+
+    // OBR-4 - Universal Service Identifier
+
+    // OBR-4.1 - Identifier
+    public static String getOBR4_1Value(ServiceRequest serviceRequest) {
+        CodeableConcept cc = serviceRequest.getCode();
+        if (cc == null || cc.getCoding().isEmpty()) {
```

Comment:
Can cc.getCoding() ever be null? If so we may need to check that

Reply:
I don't think it can ever be null. If it doesn't have Coding, it will be empty not null